src/engine/specification.md

```diff
@@ -44,50 +43,14 @@ This structure maps on the [`ExecutionPayload`](https://github.com/ethereum/cons
 - `blockHash`: `DATA`, 32 Bytes
 - `transactions`: `Array of DATA` - Array of transaction objects, each object is a byte list (`DATA`) representing `TransactionType || TransactionPayload` or `LegacyTransaction` as defined in [EIP-2718](https://eips.ethereum.org/EIPS/eip-2718)
 
-## Core
-
-### engine_preparePayload
+### PayloadAttributes
 
-#### Parameters
-1. `Object` - The payload attributes:
-- `parentHash`: `DATA`, 32 Bytes - hash of the parent block
+This structure contains the attributes (in addition to the other parameters contained in `engine_forkchoiceUpdated`) required to initiate a payload build process. The fields are encoded as follows:
```

Comment:
```suggestion
This structure contains the attributes required to initiate a payload build process. The fields are encoded as follows:
```
AFAICT, this structure doesn't contain any params from `engine_forkchoiceUpdated`? It's just an optional additional parameter?

Reply:
How about the following?

```
This structure contains the attributes required to initiate a payload build process in the context of an `engine_forkchoiceUpdated` call. The fields are encoded as follows:
```

Reply:
That sounds great!